make-4.3-patches/initial_port.patch

```diff
@@ -666,18 +666,21 @@ index 525f292..ee6e14d 100644
  /* Used to signal read() that a SIGCHLD happened.  Always CLOEXEC.
     If we use pselect() this will never be created and always -1.
 diff --git a/tests/test_driver.pl b/tests/test_driver.pl
-index f18e785..2ca84e0 100644
+index f18e785..46544a5 100644
 --- a/tests/test_driver.pl
 +++ b/tests/test_driver.pl
-@@ -382,6 +382,7 @@ sub get_osname
-     chdir (".ostest") or &error ("Couldn't chdir to .ostest: $!\n", 1);
-   }
- 
-+print $short_filenames;
```

Comment:
should this be kept or should it be removed?
Why doesn't Linux need LIBPATH too? Do they use LD_LIBRARY_PATH?